staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1alpha1/types.go

```diff
@@ -167,3 +167,41 @@ type TracingConfiguration struct {
 	// Defaults to 0.
 	SamplingRatePerMillion *int32 `json:"samplingRatePerMillion,omitempty" protobuf:"varint,2,opt,name=samplingRatePerMillion"`
 }
+
+// FlowControlConfig holds priority and fairness parameters.
+type FlowControlConfig struct {
```

Comment:
I really hate using the term "flow control" for this feature, and am not happy about more such usage appearing in the API.

This feature has a name, API Priority and Fairness.  We should be consistently using one name.

The term "flow control" is broad and misleading.  I did a web search for "flow control".  The first hit was https://en.wikipedia.org/wiki/Flow_control_(data) .  As you would expect from the words, the concepts there are around controlling rate rather than concurrency.  While it is true that the two are connected, so controlling either has an effect on the other, the problem with the term "flow control" is that it suggests that rate rather than concurrency is being directly controlled.  The techniques listed on that web page do not include things like APF.

Reply:
The reason why I suggest FlowControl is consistency, e.g. with:
https://github.com/kubernetes/kubernetes/pull/111222/files#diff-207875acc7ee03f987aa626f53742c02323ed8f434fd40dfd929dffb23d3f1deR121

It's extremely misleaing when some parts are called P&F and some other parts of the same thing are called FlowControl.
Personally - I don't have very strong preference either direction, but I have a strong preference towards consistency.

Reply:
For now renamed everything to "flow control" so that we can have a single config for all related parameters.

Reply:
I also prefer consistency.  I explained why I think "flow control" is the inferior choice.  So if we are going to go for consistency, my recommendation is to unify on "API Priority and Fairness".

Reply:
Right, I should have explained my train of thought for this change in my previous comment. I do agree with your criticism of using `FlowControl` here but the code at the earlier version of this PR had "Priority & Fairness" and "Flow Control" mixed up everywhere and I just wanted to have a single version of naming in this code change. `FlowControl` is somewhat of a worse choice but is at least consistent with the previous PR. I'm still opened to changing it to APF wherever needed provided we reach a consensus first. I also believe that sticking with `FlowControl` for now and changing it later in response to https://github.com/kubernetes/kubernetes/issues/111407 is also a viable option.